contracts/0.8.25/vaults/dashboard/Dashboard.sol

```diff
@@ -184,25 +204,31 @@ contract Dashboard is NodeOperatorFee {
      * @param _etherToFund the amount of ether to be funded, can be zero
      * @return the number of shares that can be minted using additional ether
      */
-    function remainingMintingCapacityShares(uint256 _etherToFund) external view returns (uint256) {
-        uint256 totalShares = _totalMintingCapacityShares(_etherToFund);
-        uint256 liabilityShares_ = VAULT_HUB.liabilityShares(address(_stakingVault()));
+    function remainingMintingCapacityShares(uint256 _etherToFund) public view returns (uint256) {
```

Comment:
can it be like:

```solidity
totalCapacityShares = totalMintingCapacityShares();
liabilityShares = liabilityShares();

return totalCapacityShares > liabilityShares ? totalCapacityShares - liabilityShares : 0;
```

Reply:
We need to take into account `_etherToFund` here, so not really